.integration-watchlist

```diff
@@ -1,15 +1,13 @@
 # run integration tests if files in these directories change
 demos/serverless/
-demos/browser/app/meeting/
-demos/browser/app/meetingV2/
+demos/browser
```

Comment:
this includes the app checker too now, right?

Reply:
Yeah thats why we remove the app level and just include everything in the demos/browser except README file.